packages/next/src/generators/application/lib/create-application-files.ts

```diff
@@ -52,6 +52,5 @@ export function createApplicationFiles(host: Tree, options: NormalizedSchema) {
   if (options.js) {
     host.delete(`${options.appProjectRoot}/index.d.ts`);
     toJS(host);
-    updateTsConfigsToJs(host, { projectRoot: options.appProjectRoot });
```

Comment:
all changes that are done within this function are either already in place for Next.js apps, or we don't want to perform them, due to Next.js apps not following the `tsconfig.app.json` convention